.circleci/config.yml

```diff
@@ -101,7 +167,24 @@ jobs:
             cd ops
             go run ./cmd/print_staging_report/main.go
 
+  check-codegen-changes:
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - checkout-with-mise
+      - run:
+          name: Checks if running codegen causes changes
+          command: |
+            just check-for-codegen-changes
+
 workflows:
+  scheduled-check-codegen-changes:
```

Comment:
Is this setup to notify somewhere on slack when it fails?

Reply:
No